src/processor/map/map_simple.cpp

```diff
@@ -57,6 +58,16 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapDetachDatabase(
         getOperatorID(), std::move(printInfo));
 }
 
+void mapExportDatabaseHelper(PhysicalOperator* physicalOperator,
+    const std::shared_ptr<std::atomic<bool>>& parallelFlag) {
+    for (unsigned int i{}; i < physicalOperator->getNumChildren(); ++i) {
+        if (physicalOperator->getChild(i)->getOperatorType() == PhysicalOperatorType::COPY_TO) {
```

Comment:
Except the first child, all other children must be `COPY_TO`, so you can assume all child operators starting from idx1 should be `copy_to`.